### PR TITLE
Add initial CSS guides

### PR DESCRIPTION
**Pull request by federicocastanon**
Added a section for CSS with basic content that needs to be expanded upon

### CODE REVIEW

**Review by raymall — CHANGES_REQUESTED** [review]
- We use a mobile-first approach.
- I think we don't need to mention Gulp.js
- We use SASS
- We recommend responsive scaling

**Comment by federicocastanon** [comment]
> * We use a mobile-first approach.
> * I think we don't need to mention Gulp.js
> * We use SASS
> * We recommend responsive scaling

Addressed

File: CSS/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# CSS
+
+- Use [viewport](https://www.w3schools.com/css/css_rwd_viewport.asp)  
+- Fixed number of pixels should be avoided whenever possible
+- Use percentage (%) not fixed sizes (px,em,pt)
+- We use a mobile-first approach
+- Use aspect ratio
+- We use [BEM Methodology](https://getbem.com/introduction/)
+- We use [SASS](https://sass-lang.com/)
+- We recommend responsive scaling
```
Comment (illarionvk) [inline CSS/README.md:10]: Unclear recommendation, let's revisit it in the next round.

```suggestion
```

**Review by illarionvk — CHANGES_REQUESTED** [review]
Plese rename `CSS` folder to `css` because all folder names should be lowercase in `kebab-case` format.

File: CSS/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# CSS
+
+- Use [viewport](https://www.w3schools.com/css/css_rwd_viewport.asp)
```
[the remark this answers is not in the document]
Comment (federicocastanon) [reply CSS/README.md:3]: What about [this one](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag) from Mozilla?

**Comment by federicocastanon** [comment]
@illarionvk 's comments have been addressed

**Review by illarionvk — APPROVED** [review]

**Pull request merged**